### DISCUSSION TITLE
External Data Option 4 - Use OPA's API

### DISCUSSION THREAD

**Discussion by Ronnie-personal** [open-policy-agent/community#375] [OPA and Rego]
I'm reading https://www.openpolicyagent.org/docs/latest/external-data/#option-4-push-data and I would like to understand more about the OPA's API mentioned in the document.
Do you happen to have examples for step B calling OPA's API to update or add external data to OPA server at run time?

I guess the OPA's API could be policy API or Data API?
https://www.openpolicyagent.org/docs/latest/rest-api/#create-or-overwrite-a-document
https://www.openpolicyagent.org/docs/latest/rest-api/#create-or-update-a-policy

Thanks!

**Comment by srenatus** [comment]
Yes, you'd use the data API with PUT.
